openapi/data_repository_service.swagger.yaml

```diff
@@ -21,7 +21,7 @@ security:
   - {}
   - authToken: []
 paths:
-  '/objects/{object_id}':
+  '/objects/{scheme}{identifier}/{object_id}':
```

Comment:
So if this is optional, you just want to add another endpoint rather than adding in {schema}{identifier}, right?  I don't think OpenAPI has a way of adding in an optional part of the URL path, correct?

Reply:
Actually -- since the DRS server itself never has to process the prefix, why would we even include it in the DRS API? Why can't we leave DRS alone, and just add new rules for how DRS clients resolve DRS URI's that include prefixes?